### DISCUSSION TITLE
Brainstorming Meeting

### DISCUSSION THREAD

**Discussion by ChaitanyaD20** [airavata-courses/DCoders#17] [General]
Brainstorming meeting notes,
We have all the PRs raised and we are good for our milestone-1. I would like to take a few takeaways from this milestone.
Takeaways,
1. Big learning curve.
2. Working as a team and supporting team members.
3. Understanding what team leads/managers go through during the deadlines.
4. Technical learning:
     - Microservice development
     - REST APIs
     - UI
     - Containerization
     - CI

Let's connect to start discussing the next milestone.

**Comment by vinayakasg18** [comment]
Thanks for the notes @ChaitanyaD20.
